Remotion/ObjectBinding/Web.ClientScript/Scripts/BocAutoCompleteReferenceValue.jquery.ts

```diff
@@ -31,10 +31,53 @@
 // Significantly modified for re-motion
 // ************************************************
 
+interface JQueryStatic { Autocompleter: any; }
+
+type Options = {
+    highlight: ((value: any) => any | boolean) | ((html: HTMLElement, termAsHtml: string) => any | boolean),
+    formatItem: ((row: JQuery) => { html: HTMLElement, class: string, isAnnotation: boolean }) | ((data: any, i: number, max: number, value: any, term: string) => { html: HTMLElement, class: string, isAnnotation: boolean }) | ((data: any, i: number, max: number, value: any, term: string) => { html: HTMLElement, class: string, isAnnotation: boolean })
+    formatMatch: (rawValue: any, i: number, length: number) => boolean | string,
+    serviceUrl: string,
+    serviceMethodSearch: string,
+    serviceMethodSearchExact: string,
+    data: any,
+    informationPopUpID: string,
+    dropDownButtonID: null,
+    clearRequestError: () => void,
+    inputClass: string,
+    selectionUpdateDelay: number,
+    dropDownRefreshDelay: number,
+    dropDownDisplayDelay: number,
+    autoFill: Nullable<(query: string, sValue: string) => void>,
+    nullValue: any,
+    isAutoPostBackEnabled: boolean,
+    extraParams: Dictionary<string | number> ,
+    params: Dictionary<any>,
+    searchStringValidationParams: { inputRegex: { test: (x: any) => boolean }, dropDownTriggerRegex: { test: (x: any) => boolean }, getDropDownSearchString: (x: string) => string, dropDownTriggerRegexFailedMessage: string },
+    noDataFoundMessage: string,
+    selectFirst: (val: any, q: Nullable<string>) => any,
+    max: number,
+    handleRequestError: (err: Sys.Net.WebServiceError) => void,
+    parse: (result: any) => any,
+    formatResult: (row: string[], row2?: string) => any
```

Comment:
`data` and `value` should probably get an actual type. Needs checking what type this would be, but I think we already have them defined in the BocAutoCompleteReferenceValue.ts file